src/ReaderTaskEither.ts

```diff
@@ -187,6 +187,12 @@ export const leftIO: <E, R, A = never>(me: IO<E>) => ReaderTaskEither<R, E, A> =
  */
 export const fromReader: FromReader3<URI>['fromReader'] = rightReader
 
+/**
+ * @category constructors
+ * @since 3.0.0
+ */
+export const fromReaderTask: <R, A, E = never>(ma: ReaderTask<R, A>) => ReaderTaskEither<R, E, A> = rightReaderTask
```

Comment:
this is just an alias, I think we can drop it

Reply:
It's removed!

Reply:
I forgot to update the doc. I'm doing this quickly

Reply:
it's done!